qiskit/dagcircuit/collect_blocks.py

```diff
@@ -318,15 +320,18 @@ def split_block_into_layers(block):
 
 class BlockCollapser:
     """This class implements various strategies of consolidating blocks of nodes
-     in a DAG (direct acyclic graph). It works both with the
-    :class:`~qiskit.dagcircuit.DAGCircuit` and
-    :class:`~qiskit.dagcircuit.DAGDependency` DAG representations.
+    in a DAG (direct acyclic graph). It works both with
+    the :class:`~qiskit.dagcircuit.DAGCircuit`
+    and :class:`~qiskit.dagcircuit.DAGDependency` DAG representations.
```

Comment:
Ha, it's the extra space that was causing the errors, not the leading with `:class:`.  I was looking at the wrong docstring when I was trying to help you debug  - I'd not noticed the difference between `BlockCollector` and `BlockCollapser` haha.